src/transformers/image_transforms.py

```diff
@@ -418,15 +405,10 @@ def center_crop(
     """
     requires_backends(center_crop, ["vision"])
 
-    if isinstance(image, PIL.Image.Image):
-        warnings.warn(
-            "PIL.Image.Image inputs are deprecated and will be removed in v4.26.0. Please use numpy arrays instead.",
-            FutureWarning,
-        )
-        image = to_numpy_array(image)
-        return_numpy = False if return_numpy is None else return_numpy
-    else:
-        return_numpy = True if return_numpy is None else return_numpy
+    if return_numpy is not None:
+        warnings.warn("return_numpy is deprecated and will be removed in v.4.33", FutureWarning)
+
```

Comment:
In the `center_crop` method, it used to accept and optionally return PIL.Image.Image objects. This was to keep backwards compatibility with the previous cropping logic.

The method had a warning about accepting PIL.Image.Image's as input, but not about removing the `return_numpy` argument. We remove the accepting PIL logic and add a warning for removing `return_numpy` here